gapic/utils/case.py

```diff
@@ -45,3 +45,17 @@ def to_snake_case(s: str) -> str:
 
     # Done; return the camel-cased string.
     return s.lower()
+
+def to_camel_case(s: str) -> str:
+    '''Convert any string to camel case.
+
+    This is provided to templates as the ``camel_case`` filter.
+
+    Args:
+        s (str): The input string, provided in snake case.
+    
+    Returns:
+        str: The string in camel case with the first letter unchanged.
+    '''
+    items = s.split('_')
+    return items[0] + "".join([x.capitalize() for x in items[1:]])
```

Comment:
Nit: no need to make a list as the argument to join. We could replace the square brackets with parentheses to make it a generator expression, but there's a minor syntactic optimization where if a generator expression is the sole function argument you can remove the parens.
```python
join(x.capitalize() for x in items[1:])
```